config/routes.rb

```diff
@@ -5,6 +5,9 @@
   root 'users#index'
 
   resources :users, only: %i[index show] do
-    resources :posts, only: %i[index show]
+    resources :posts, only: %i[index show new create] do
+      resources :comments, only: %i[create]
```

Comment:
- [ ] As required for the project, once you have added the `New` action with its expected code, kindly ensure that you also add its `route` in the routes.rb file. That means you should update the current routes for your comments to match the expected result.